src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -1344,42 +1344,49 @@ private void WriteMessageAligned(string message, bool prefixAlreadyWritten, int
             {
                 int adjustedPrefixWidth = _prefixWidth + prefixAdjustment;
 
-                // The string may contain new lines, treat each new line as a different string to format and send to the console
-                string[] nonNullMessages = SplitStringOnNewLines(message);
-                for (int i = 0; i < nonNullMessages.Length; i++)
+                if (Traits.Instance.EscapeHatches.DoNotOptimizeConsoleLogger || !ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_0))
```

Comment:
Not sure why this was diffed so badly, but whatever is inside this `if` is identical to the old code, for ChangeWave purposes.

Reply:
Add `&w=1` to the URL, and it looks great:
![image](https://user-images.githubusercontent.com/12969783/141355583-d2fe8343-1868-4a6d-9b3b-1c1830a2673d.png)

Reply:
Found it:
![image](https://user-images.githubusercontent.com/25249058/141375673-88f2ac24-4b96-4dfc-b181-edfd82873877.png)